test/library/packages/ZMQ/pair.chpl

```diff
@@ -121,6 +128,14 @@ proc Worker() {
     writeln("val = ", val);
   }
 
+  // Bytes
+  {
+    var val = b"The quick brown fox...";
```

Comment:
What if you throw a non-UTF8 byte into this literal?

Reply:
Good question.. I hope it works :) I'll make that change to the test